### PR TITLE
Create remand SC for noncomp decision review with no end products

### PR DESCRIPTION
**Pull request by leikkisa**
connects #10171

### CODE REVIEW

File: spec/support/intake_helpers.rb [hunk 1]
```diff
@@ -397,31 +397,19 @@ def setup_prior_decision_issues(veteran, benefit_type: "compensation")
                              rating_request_issue.contested_decision_issue_id])
   end
 
-  def setup_prior_claim_with_payee_code(appeal, veteran, prior_payee_code = "10")
-    prior_supplemental_claim = create(
```
Comment (leikkisa) [inline spec/support/intake_helpers.rb]: If someone selects a claimant on a noncomp decision review, and that gets remanded, we just need that claimant to be on a previous end product (not on a previous remand supplemental claim).

File: spec/models/tasks/decision_review_task_spec.rb [hunk 2]
```diff
@@ -12,13 +16,21 @@
...
 
   describe "#complete_with_payload!" do
+    before do
+      setup_prior_claim_with_payee_code(hlr, veteran, "00")
+    end
+
     let(:veteran) { create(:veteran) }
-    let(:hlr) { create(:higher_level_review, veteran_file_number: veteran.file_number, benefit_type: benefit_type) }
+    let(:hlr) do
+      create(:higher_level_review, number_of_claimants: 1,
```
Comment (pkarman) [inline spec/models/tasks/decision_review_task_spec.rb]: the layout here is a little weird? I find it easiest to read when left aligned as much as possible.

Comment (leikkisa) [reply spec/models/tasks/decision_review_task_spec.rb]: Ah yes, that does look weird. Thanks!

**Comment by leikkisa** [comment]
Oh, I shouldn't validate this one, but I was trying to validate another one, and created: HLR 1326 with an education issue, and made a decision on it, and it created the remand SC 451

**Pull request merged**